neo4j_utils/_driver.py

```diff
@@ -1010,7 +1009,7 @@ def drop_indices_constraints(self):
         Requires the database to be empty.
         """
 
-        neo4j_version = _n4jversion.Neo4jVersion()
+        neo4j_version = self.get_neo4j_version()
```

Comment:
this is indeed much better -- so the version is not necessary before connection?